servers/rendering/dummy/storage/mesh_storage.cpp

```diff
@@ -87,6 +87,13 @@ void MeshStorage::_multimesh_set_buffer(RID p_multimesh, const Vector<float> &p_
 	memcpy(cache_data, p_buffer.ptr(), p_buffer.size() * sizeof(float));
 }
 
+RID MeshStorage::_multimesh_get_buffer_rd_rid(RID p_multimesh) const {
```

Comment:
This should just be implemented in the header file and only ever return RID(). See functions like `_multimesh_instance_get_color()`

Reply:
Roger that, I rebased, synced with master and got that change in.